README.md

```diff
@@ -231,6 +264,88 @@ By default, `beekeeper-scheduler-apiary` listens on port 8080, `beekeeper-path-c
 | `scheduler-delay-ms`                | No       | Amount of time (in milliseconds) between consecutive cleanups. Default value is `300000` (5 minutes after the previous cleanup completes). |
 | `Metastore-uri`                     | Yes      | URI of the Hive Metastore where tables to be cleaned-up are located. |
 
+## Beekeeper-API
+
+Beekeeper also has an API which provides read access to the Beekeeper database and allows you to see what metadata and paths are currently being managed.
+
+The user can manually enter a database and a table name and check whether this table has been successfully registered in Beekeeper along with things like the current status of the table, the date and time it will be deleted, the current cleanup delay... etc.
+
+It currently supports two endpoints; one for the expired metadata and another one for the unreferenced paths.
+
+As well as supporting all [standard actuator endpoints](https://docs.spring.io/spring-boot/docs/current/reference/html/production-ready-endpoints.html), the Beekeeper-API also supports the `swagger` endpoint (see the [Swagger documentation](https://swagger.io/docs/specification/about/)), which provides a visual documentation of the structure of the API, making it easy for the user to explore its capabilities. This is a good start if it's the first time you use the API. You can access it at this url:
+
+    http://<host>/swagger-ui.html
+
+For the two main endpoints, the base url (will be referred to as `<base-url>` in the following sections) is
+
+    http://<host>/api/v1
+
+and the rest of the url will depend on which endpoint you want to access (see [Expired metadata endpoint](#expired-metadata-endpoint) and [Unreferenced paths endpoint](#unreferenced-paths-endpoint)).
+
+It also supports different filters (see [filtering section](#filtering)).
+
+### Expired metadata endpoint (`GET /metadata`)
+
+This endpoint will return the TTL configuration of all expired partitions that are going to be deleted (or have been deleted) in a specific table. If it is unpartitioned it will just show one object; the table.
+
+    <base-url>/database/{databaseName}/table/{tableName}/metadata
+
+ where `{databaseName}` and `{tableName}` must be replaced by the database and table name the user wants to search for. So for example, if they wanted to check a table called `my_cool_table` in the database `my_cool_database`, they would go to 
+ 
+    <base-url>/database/my_cool_database/table/my_cool_table/metadata
+    
+The API will display all the partitions in that table unless it is unpartitioned, in that case it will show only one object; the table. If you want to check only the table object without all of its individual partitions, you'll have to search for the one with the variable `"partitionName"=null` as such:
```

Comment:
You should change the second person style in this writing and adopt the third person to make it look more generic and professional; unless the entire document is written in the second person in which case is OK to keep it consistent.
Example: If you want...